client/tsconfig.json

```diff
@@ -23,7 +23,7 @@
 		{ "path": "../packages/media-library" },
 		{ "path": "../packages/react-i18n" }
 	],
-	"include": [ "**/*" ],
+	"include": [ "**/*", "../build-tools/**" ],
```

Comment:
Why does this _tsconfig.json_ care about a sibling directory that doesn't even contain any TypeScript files?

I know very little about configuring TypeScript and don't use VScode myself, so I'd like to learn more about what changes like this do.

Reply:
TS compiler needs to have access to all files (TS or not), specially because this TS project is configured as `allowJs: true` (comes from `@automattic/calypso-build/tsconfig`).

Because `build-tools` is referenced from `client/webpack.config.js` _and_ it lives outside `rootDir` (configured a few lines above), it won't be matched by `**/*` so we need to declare it explicitly.

Reply:
Is this all because there is `require( 'webpack.config.js' )` in the dev server code? If the config wasn't used by the app, would we still need to tell TS about it?

Reply:
Is because `./client/webpack.config.js` is inside this TS project (it is matched by `**/*`) but requires files outside that project (`require('../build-tools/...')`. Having a file that requires external files makes VS Code unhappy:

![image](https://user-images.githubusercontent.com/975703/97135088-9e435480-174f-11eb-8290-ca6e3b5fde17.png)